README.adoc

```diff
@@ -486,56 +411,32 @@ InventoryResource.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
 ----
 
-In order to access the Tracer, the [hotspot=inject file=0]`@Inject` annotation from the Contexts and Dependency Injections API injects the Tracer into a bean. 
+To access the Tracer, inject it into a bean by using the [hotspot=inject file=0]`@Inject` annotation from the Contexts and Dependency Injections API.
 
-Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` calls the `system` service, the `spanBuilder()` and `startSpan()` Tracer APIs create and start a span called [hotspot=getPropertiesSpan file=0]`GettingProperties`.
+Before the [hotspot=manager hotspot=getSystem file=0]`InventoryManager` to call the `system` service, create and start a span called [hotspot=getPropertiesSpan file=0]`GettingProperties` by using the `spanBuilder()` and `startSpan()` Tracer APIs.
```

Comment:
Sentence needs rewritten, the grammar is all wrong.

Maybe to this?

Before the `InventoryManager` calls the `system` service, it creates and starts a span called `GettingProperties` by using the `spanBuilder()` and `startSpan()` Tracer APIs.